test/unit/image/loading.js

```diff
@@ -7,7 +7,7 @@ var testImageRender = function(file, sketch, callback) {
   var p = sketch.pixels;
   var ctx = sketch;
 
-  sketch.clear();
+  sketch.drawingContext.clearRect(0,0,this.width, this.height);
```

Comment:
this shouldn't be necessary because the color/ files will always be packaged with core